.github/workflows/test_e2e_deploy_release.yml

```diff
@@ -108,10 +108,6 @@ jobs:
       - id: test-result
         name: Set test result variable
         run: echo 'immediately-close=${{ needs.test-deploy.result != 'success' && 'true' || 'false' }}' >> "$GITHUB_OUTPUT"
-      - name: Check token
-        run: gh auth status
-        env:
-          github-token: ${{ secrets.GH_UPDATE_NEXT_WORKFLOW_TRIGGER }}
```

Comment:
It's not unnecessary. It is a preflight check so that it's clearer when the token is expired. We previously didn't have this and wasted time waiting on script execution when it could've been clearly early what was wrong.

If the workflow fails because of this check, it'll also fail `trigger-front-sync`

Reply:
If that's not the case, we can remove any mention of `secrets. GH_UPDATE_NEXT_WORKFLOW_TRIGGER`.

Reply:
https://github.com/vercel/next.js/pull/81166/